test/cluster/mv/test_mv_building.py

```diff
@@ -113,3 +113,26 @@ async def test_view_building_with_tablet_move(manager: ManagerClient, build_mode
 
         for view in views:
             await wait_for_view(cql, view, len(servers))
+
+# While view building is in progress, drop the view and change the schema
+# of the base table. The view table's state may become inconsistent with
+# the base table because they are detached.
+@pytest.mark.asyncio
+async def test_view_building_during_drop_view(manager: ManagerClient):
+    server = await manager.server_add()
+    cql = manager.get_cql()
+    await manager.api.enable_injection(server.ip_addr, "view_builder_consume_end_of_partition_delay", one_shot=True)
+
+    await cql.run_async(f"CREATE KEYSPACE ks WITH replication = {{'class': 'NetworkTopologyStrategy', 'replication_factor': 1}}")
+    await cql.run_async(f"CREATE TABLE ks.tab (p int, c int, PRIMARY KEY (p, c))")
+    await cql.run_async("INSERT INTO ks.tab (p,c) VALUES (123, 1000)")
+
+    await cql.run_async("CREATE INDEX idx1 ON ks.tab (c)")
+    # wait for view building to start
+    await asyncio.sleep(1)
```

Comment:
Would have been nicer to find a better way to wait for view building to start (e.g., look at the view building progress system table) so that you don't need to sleep a whole second.
But I don't want to insist given that other cluster tests aren't so speedy either :-(